noxfile.py

```diff
@@ -416,6 +418,9 @@ def cover(session):
     test runs (not system test runs), and then erases coverage data.
     """
     session.install("coverage", "pytest-cov")
+
+    session.run("python", "-m", "pip", "freeze")
```

Comment:
Please check `owlbot.py` to ensure that all the changes we want in the `noxfile` are captured correctly in `owlbot`.

See @chalmerlowe to talk about using the `owlbot` docker container to local testing, if you are not already familiar with the Google `owlbot docker container.